transit/rest_api/forms/shipment/shipment_details.py

```diff
@@ -22,6 +22,13 @@ def to_representation(self, value):  # noqa: WPS122
         return list(relevant_line_items)
 
 
+class ShipmentOrderMappingOrderNumbersReadonlyField(CharField):
```

Comment:
It'll be better if we combine this with `ShipmentOrderMappingCustomerNamesReadonlyField` 
e.g. you can change CharField to Dict field which would return something as: 

```json 
{"orders": [
{"id": 2, "name": "product"}
]}
```

no reason to keep them in separate serializers as they are regarding single object realtion.

Reply:
ok, sure. I create ticket to do this change in the future https://sd-transit.atlassian.net/browse/TRAN-150